### PR TITLE
EZP-29140: User bundle

### PR DESCRIPTION
**Pull request by ViniTou**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29140
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Move and deprecate things:
https://github.com/ezsystems/ezplatform-user/pull/1

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by adamwojs** [comment]
Just to be safe it would be great to have confirmation that everything is works from CI. You can achive it by temporary configure Travis to use specyfic branch in meta repository:  https://github.com/ezsystems/ezplatform-admin-ui/blob/master/composer.json#L39 @mnocon knows details :wink:

File: src/bundle/Resources/config/routing.yml [hunk 1]
```diff
@@ -598,49 +598,6 @@ ezplatform.user.delete:
...
-    path: /user/forgot-password
-    defaults:
-        _controller: 'EzPlatformAdminUiBundle:User\UserForgotPassword:userForgotPassword'
-
-ezplatform.user.forgot_password.login:
-    path: /user/forgot-password/login
-    defaults:
-        _controller: 'EzPlatformAdminUiBundle:User\UserForgotPassword:userForgotPasswordLogin'
-
-ezplatform.user.reset_password:
```
[the remark this answers is not in the document]
Comment (ViniTou) [reply src/bundle/Resources/config/routing.yml]: moved with same names

**Comment by ViniTou** [comment]
@adamwojs 
@webhdx 
All green

**Comment by lserwatka** [comment]
Let's proceed here. Could you remove this tmp commit for travis? I will merge it afterwards.

**Comment by ViniTou** [comment]
@lserwatka 
done. 
I also put up PR to metarepo:
https://github.com/ezsystems/ezplatform/pull/361

**Pull request merged**